### DISCUSSION TITLE
Problem with a Hijacking Page on Facebook - already tried to find Facebook issues page, but there is no category for help there

### DISCUSSION THREAD

**Discussion by Theeperiwinkle** [uBlockOrigin/uAssets#26108] [General]
I have a page that keeps hijacking my Candy Crush game.  All of a sudden a reset password page pops up and I can't use the go back arrow to get rid of it.  I have to reload the page and it ruins my game.  I'm trying to file a report with Facebook, but their help is less than simple to follow.  I don't know how to block the page that keeps popping up.

Here is the link:  https://www.facebook.com/recover/initiate/?is_from_lara_screen=1

**Comment by Yuki2718** [comment]
How is this related to uBO?
